.gitignore

```diff
@@ -5,3 +5,4 @@
 *~
 .project
 .pydevproject
+wsgi/media/**
```

Comment:
for ex: `/media/photologue/photos/**.jpg` is the look up for any image imported from photologue.